lib/components/app/desktop-nav.tsx

```diff
@@ -46,9 +46,29 @@ const DesktopNav = ({
   popupTarget,
   setPopupContent
 }: Props) => {
-  const { branding, persistence, title = DEFAULT_APP_TITLE } = otpConfig
+  const {
+    branding,
+    persistence,
+    title = DEFAULT_APP_TITLE,
+    brandClickable
+  } = otpConfig
   const showLogin = Boolean(getAuth0Config(persistence))
 
+  const BrandingElement = brandClickable ? 'a' : 'div'
```

Comment:
Maybe using a `<span>` instead of `<div>` would allow to move all the CSS below to `commonStyles`.

Reply:
I used `div` because it was what the element was previously using. `span` doesn't seem to affect anything, but I'm still a bit nervous about the change and I don't see which styles can be retired